src/main/java/lotto/controller/LottoController.java

```diff
@@ -14,30 +15,69 @@
 public class LottoController {
 
     public void start() {
-        LottoService lottoService = new LottoService(new RandomLottoMachine());
-        Ticket ticket = buyLotto();
+        final Lotteries lotteries = new Lotteries();
+        final Money money = getMoney();
+        final Ticket ticket = new Ticket(money, manualBuyTicket());
+
+        generateLotto(lotteries, ticket);
```

Comment:
Lotteries를 미리 만들어서 메소드로 넘겨주지 않고 아래처럼 generateLotto로부터 반환 받아도 될 것 같아요.
```suggestion
        final Lotteries lotteries = generateLotto(ticket);
```